### DISCUSSION TITLE
Welcome to Laureate Discussions!

### DISCUSSION THREAD

**Discussion by subhoghoshX** [subhoghoshX/laureate#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Karthik-315** [comment]
Hello All! Hope you are having a great day! 

I am Karthik, a front-end developer, and an Open Source enthusiast. I am really glad to be part of this OS project and to be working with all of you. Looking forward to contributing to this project, in any way possible. 

Cheers!
